### DISCUSSION TITLE
Modeline for persistent marks or last cursor position

### DISCUSSION THREAD

**Discussion by avidseeker** [vim/vim#15999] [Q&A]
viminfo (or nvim's shada) supports this, but it's not portable. Furthermore, it might be desirable to manually set the position. (e.g: `vim book.txt` and user wants to open at certain section).

Can modeline be used to save marks (or set them at vim startup)?

**Comment by chrisbra** [comment]
No modelines intententionally are only there to set some options. You may want to check `:h last-position-jump`
